src/src/com/microsoft/aad/adal/AuthenticationConstants.java

```diff
@@ -212,6 +212,8 @@
         
         public static final String BROKER_VERSION = "broker.version";
         
+        public static final String BROKER_PROTOCOL_VERSION = "v2";
```

Comment:
nit: consider removing 'v', then you can remove the equalsIgnoreCase and just do equals